packages/components/src/config/variables.ts

```diff
@@ -8,6 +8,15 @@ export const SCREEN_SIZE = {
     XL: '1200px', // extra Large laptops/desktops
 } as const;
 
+export const SCREEN = {
```

Comment:
I would give it better name. 

Usage `${variables.SCREEN.ABOVE_TABLET} {` could be confusing. `MEDIA_QUERY_SCREEN` or something like that would be nice.

Reply:
done